Cargo.toml

```diff
@@ -25,3 +25,6 @@ strum_macros = "0.24.2"
 [lib]
 name = "gdcm_rs"
 path = "src/lib.rs"
+
+[features]
+disable-charls = []
```

Comment:
```suggestion
[features]
default = ["charls"]
charls = []
```

Reply:
Good catch, it is much cleaner this way

Reply:
I also updated the README.md to document the flag